shady-css-parser.html

```diff
@@ -0,0 +1 @@
+<script src="dist/shady-css.js"></script>
```

Comment:
I hate all JS packing setups, but the one I've been using lately is to put the amd/globals/bower version in the root, say as just `shady-css.js` so that it's slightly cleaner from bower. ymmv